pyalertme/zbnode.py

```diff
@@ -296,8 +328,9 @@ def __init__(self, serial, callback=None):
 
         # Type Info
         self.type = 'ZBNode'
-        self.version = 12345
-        self.manu = 'PyAlertMe'
+        self.hwMajorVersion = 123
```

Comment:
Minor:
Perhaps it would be better to set these defaults to something a little more appropriate like:
+        self.hwMajorVersion = 001
 +        self.hwMinorVersion = 01
? 
Fine to leave as is for now :-)

Good work splitting the Major / Minor version

Reply:
Or... do you think it would be appropriate to simply replace this with the NodeId which you have worked out?

Reply:
These are fine for now, but I have thought of changing them to mimic the real hub, as the smartplug / etc may expect specific stuff.